config/notifiers.go

```diff
@@ -67,6 +67,18 @@ var (
 		Fallback:  `{{ template "slack.default.fallback" . }}`,
 	}
 
+	// DefaultHipchatConfig defines default values for Hipchat configurations.
+	DefaultHipchatConfig = HipchatConfig{
+		NotifierConfig: NotifierConfig{
+			VSendResolved: false,
```

Comment:
Slack being true is an oversight, I've already sent a PR to correct it.

Resolved notifications are almost pure spam. Unfortunately some people come believe that they're essential in the same way that email alert spam is believed to be essential. To discourage this, the default should be to not send them so that our users are less likely to get caught in a downwards spiral of being overloaded by notifications for which they take no action.

It should only be enabled by default for systems where they do something meaningful with it, such as Pagerduty and friends. Otherwise it should be strongly discouraged.